cypress/integration/projects/access/project-access.spec.ts

```diff
@@ -0,0 +1,147 @@
+/// <reference types="cypress" />
+
+import {
+    PA_ASSIGN_BUTTON_ID,
+    PA_ASSIGN_CREATE_ID,
+    PA_EDIT_BUTTON_ID,
+    PA_REMOVE_BUTTON_ID,
+    PA_ROLE_ID,
+    PA_USERS_GROUPS_ID,
+    PA_USERS_GROUPS_TITLE_ID,
+} from '../../../../src/utils/testIds';
+
+export {};
+const baseUrl = Cypress.config().baseUrl;
+const randomId = String(Math.random()).split('.')[1];
+const groupAndProjectName = `group-e2e-${randomId}`;
+const userName = `user-e2e-${randomId}`;
+const groupIds: any[] = [];
+const userIds: any[] = [];
+
+// Disable all active splash pages by visiting them.
+const disableActiveSplashScreens = () => {
+    cy.visit(`/splash/operators`);
+};
+
+describe('project-access', () => {
+    before(() => {
+        disableActiveSplashScreens();
+        cy.login();
+        for (let i = 1; i <= 2; i++) {
+            const name = `${i}-${userName}`;
+            cy.request('POST', `${baseUrl}/api/admin/user-admin`, {
+                name: name,
+                email: `${name}@test.com`,
+                sendEmail: false,
+                rootRole: 3,
+            })
+                .as(name)
+                .then(response => {
+                    const id = response.body.id;
+                    userIds.push(id);
+                    cy.request('POST', `${baseUrl}/api/admin/groups`, {
+                        name: `${i}-${groupAndProjectName}`,
+                        users: [{ user: { id: id }, role: 'Owner' }],
```

Comment:
Should be adapted once https://github.com/Unleash/unleash/pull/1905 gets merged.
```suggestion
                        users: [{ user: { id: id }],
```

Reply:
Removed